src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -566,6 +586,9 @@ void DlgPrefDeck::slotApply() {
     m_pConfig->setValue(ConfigKey("[Controls]", "AllowTrackLoadToPlayingDeck"),
                         !m_bDisallowTrackLoadToPlayingDeck);
 
+    m_pConfig->setValue(ConfigKey("[Controls]", "CloneDeckOnLoadDoubleTap"),
+                        m_bCloneDeckOnLoadDoubleTap);
```

Comment:
Why not read the checkbox here and omit the m_bCloneDeckOnLoadDoubleTap vairable?

Reply:
because we do it via variables for all the other settings?

Reply:
It is maybe we can improve this as well one day. It jumps into my eyes, that the syncing of the checkbox and the variable has no value.